virtcontainers/ich.go

```diff
@@ -0,0 +1,813 @@
+// Copyright (c) 2019 Ericsson Eurolab Deutschland GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"bufio"
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+	"fmt"
+	"net"
+	"syscall"
+
+ 
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/pkg/errors"
+	persistapi "github.com/kata-containers/runtime/virtcontainers/persist/api"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/kata-containers/runtime/virtcontainers/store"
+	"github.com/kata-containers/runtime/virtcontainers/types"
+	"github.com/kata-containers/runtime/virtcontainers/utils"
+	"github.com/kata-containers/runtime/virtcontainers/device/config"
+	
+	govmmIch "github.com/kata-containers/runtime/virtcontainers/pkg/intel/govmm/ich"
+	
+)
+
+//
+// Constants and type definitions related to cloud hypervisor
+//
+
+type ichState uint8
+
+const (
+	ichNotReady ichState = iota
+	ichReady
+)
+
+const (
+	ichTimeout            = 10
+	ichSocket             = "ich.sock"
+	vfsdSocket            = "virtiofsd.sock"
+	apiSocket             = "cloud-hypervisor.9166"
```

Comment:
What is `9166`? Can this be simply `cloud-hypervisor.sock`?

Reply:
it will be removed.